shared_model/interfaces/iroha_internal/transaction_batch_factory.hpp

```diff
@@ -33,9 +33,11 @@ namespace shared_model {
        * batch
        * @return valid batch of transactions or string error
        */
-      FactoryResult<std::unique_ptr<TransactionBatch>>
-      virtual createTransactionBatch(
-          const types::SharedTxsCollectionType &transactions) const = 0;
+      FactoryResult<std::unique_ptr<
```

Comment:
Virtual keyword needs to be moved to the beginning, then the formatter won't do insane formatting
```
      virtual FactoryResult<std::unique_ptr<TransactionBatch>>
      createTransactionBatch(
          const types::SharedTxsCollectionType &transactions) const = 0;
```